content/en/Getting started/planning.md

```diff
@@ -42,61 +42,54 @@ The characteristics differ for each [asset type](../assets/asset-type/):
 - [Assets of multiple types](#assets-of-multiple-types)
 
-Once you’ve scoped the pentest, see how many [credits](#view-required-credits) it requires.
+Once you’ve scoped the pentest, review the required [credits](#view-required-credits), as determined by our algorithm.
 
 ### Web
 
-To scope a pentest for a Web asset, specify the number of the following characteristics of that asset:
+To scope a pentest for a Web asset, specify the number of the following characteristics of the asset that need to be tested:
 
-- [User Roles](../glossary/#user-role)
-- [Pages](../glossary/#web-page)/[Routes](../glossary/#route-software)
-
-Include those User Roles and Pages/Routes that should be in scope for this pentest. Be thorough. If you forget certain roles or pages/routes, your pentest might not cover all critical details.
-
-{{% alert title="Note" color="note" %}}
-You may not need to include _every_ user role. For example, if you have dedicated administrative roles for backups, logs, and printers, that counts as one (1) role.
-{{% /alert %}}
-
-As part of our tests for web pages, we also test the backend API endpoints frequently used to populate content on those pages.
-
-Our pentesters need to know more about your Web asset, including:
-
-- Application type, such as a page-driven website or a [single-page application](../glossary/#single-page-application)
-- Special endpoints associated with your pages
+| Scoping Parameter | Definition | Notes |
+|---|---|---|
+| [User Roles](/getting-started/glossary/#user-role) | {{% user-role %}} | <p>Enter the number of User Roles in your Web asset that need to be tested.</p><p>{{% count-user-roles %}}</p> |
+| [Pages](/getting-started/glossary/#web-page)/[Routes](/getting-started/glossary/#route-software) | <p>A Page is a hypertext document with a unique URL that a user interacts with.</p><p>A Route is a system for resource navigation in [single-page applications (SPAs)](/getting-started/glossary/#single-page-application). In SPAs that use frameworks such as Angular, React, or Ember, routes provide unique URLs to specific content within the application.</p> | Determine the type of your Web asset:<br><ul><li>**Traditional web application**. Enter the number of Pages to test.<ul><li>You may want to skip [static pages](/getting-started/glossary/#static-web-page) because security-related risks are low for them.<li>For [dynamic pages](/getting-started/glossary/#dynamic-web-page), count the number based on unique page templates.</li><li>As part of our tests for [dynamic pages](/getting-started/glossary/#dynamic-web-page), we also test the backend API endpoints frequently used to populate content on those pages.</li></ul></li><li>[Single-page application](/getting-started/glossary/#single-page-application). Enter the number of Routes to test.</li></ul>Usually, an application includes one or more routing modules or files where you can retrieve the number of pages or routes using special commands or tools. |
```

Comment:
For consistency (unless the link messes up the bolding... in which case we'd need `<b>something</b>` I guess?

```suggestion
| [Pages](/getting-started/glossary/#web-page)/[Routes](/getting-started/glossary/#route-software) | <p>A Page is a hypertext document with a unique URL that a user interacts with.</p><p>A Route is a system for resource navigation in [single-page applications (SPAs)](/getting-started/glossary/#single-page-application). In SPAs that use frameworks such as Angular, React, or Ember, routes provide unique URLs to specific content within the application.</p> | Determine the type of your Web asset:<br><ul><li>**Traditional web application**. Enter the number of Pages to test.<ul><li>You may want to skip [static pages](/getting-started/glossary/#static-web-page) because security-related risks are low for them.<li>For [dynamic pages](/getting-started/glossary/#dynamic-web-page), count the number based on unique page templates.</li><li>As part of our tests for [dynamic pages](/getting-started/glossary/#dynamic-web-page), we also test the backend API endpoints frequently used to populate content on those pages.</li></ul></li><li>[**Single-page application**](/getting-started/glossary/#single-page-application). Enter the number of Routes to test.</li></ul>Usually, an application includes one or more routing modules or files where you can retrieve the number of pages or routes using special commands or tools. |
```